agent/stats/common_test.go

```diff
@@ -128,6 +131,18 @@ func (resolver *IntegContainerMetadataResolver) ResolveContainer(containerID str
 	return container, nil
 }
 
+func validateInstanceMetrics(t *testing.T, engine *DockerStatsEngine) {
+	metadata, taskMetrics, err := engine.GetInstanceMetrics()
+	assert.NoError(t, err, "gettting instance metrics failed")
```

Comment:
should we use `require`, instead of `assert` here and in other tests?

Reply:
For this kind of test, we may not want to terminate, it still has value to continue the test.